client/components/seo/settings-upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import QueryPlans from 'components/data/query-plans';
+import Gridicon from 'components/gridicon';
+import PlanCompareCard from 'my-sites/plan-compare-card';
+import PlanCompareCardItem from 'my-sites/plan-compare-card/item';
+import TrackComponentView from 'lib/analytics/track-component-view';
+import formatCurrency from 'lib/format-currency';
+import { preventWidows } from 'lib/formatting';
+import { getFeatureTitle } from 'lib/plans';
+import { getPlanBySlug } from 'state/plans/selectors';
+import {
+	PLAN_BUSINESS,
+	FEATURE_ADVANCED_DESIGN,
+	FEATURE_CUSTOM_DOMAIN,
+	FEATURE_NO_ADS,
+	FEATURE_UNLIMITED_PREMIUM_THEMES,
+	FEATURE_UNLIMITED_STORAGE,
+	FEATURE_VIDEO_UPLOADS
+} from 'lib/plans/constants';
+
+const featuresToShow = [
+	FEATURE_UNLIMITED_STORAGE,
+	FEATURE_UNLIMITED_PREMIUM_THEMES,
+	FEATURE_CUSTOM_DOMAIN,
+	FEATURE_NO_ADS,
+	FEATURE_ADVANCED_DESIGN,
+	FEATURE_VIDEO_UPLOADS
+];
+
+const SeoSettingsNudge = ( { translate, plan = {}, upgradeToBusiness = noop } ) => {
```

Comment:
Do we really need a `noop` here? It seems reasonable that having the ability to upgrade to business is integral enough to this component that we could make it a required prop. Silently discarding it here could hide the fact that something isn't passing the right callback.